esp8266/modnetwork.c

```diff
@@ -39,6 +39,7 @@
 #include "espconn.h"
 #include "spi_flash.h"
 #include "ets_alt_task.h"
+#include "../lib/lwip/src/include/lwip/dns.h"
```

Comment:
FYI, this should be just "lwip/dns.h" (and rely on proper CC include path to be set). That's because version of lwIP included in ../lib is different from one used by esp8266, and we already have cases of differences between them. I'll fix this myself.